clients/java/src/test/java/io/camunda/zeebe/client/process/PublishMessageTest.java

```diff
@@ -63,6 +63,22 @@ public void shouldPublishMessage() {
     rule.verifyDefaultRequestTimeout();
   }
 
+  @Test
+  public void shouldPublishMessageWithDefaultTtl() {
+    // given
+    final long messageKey = 123L;
+    gatewayService.onPublishMessageRequest(messageKey);
+
+    // when
+    final PublishMessageResponse response =
+        client.newPublishMessageCommand().messageName("name").correlationKey("key").send().join();
+
+    // then
+    final PublishMessageRequest request = gatewayService.getLastRequest();
+    assertThat(request.getTimeToLive()).isEqualTo(Duration.ofHours(1).toMillis());
```

Comment:
💭 Ideally, the default ttl would be a constant we can use here to avoid adjusting magic numbers in tests if we change it in the future. But I guess it's stable enough, so 🤷

Reply:
👍  let me extract that for reuse.